.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+# Pull Request Template (adapted from [`embeddedartistry`](https://github.com/embeddedartistry/templates/blob/master/oss_docs/PULL_REQUEST_TEMPLATE.md))
+
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update (please include `[docs only]` in commit message)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+**Test Configuration**:
+* Firmware version:
+* Hardware:
+* Toolchain:
+* SDK:
+
+## Checklist (As relevant):
+
+- [ ] I have read the [Code of Conduct](CODE_OF_CONDUCT.md)
+- [ ] I have read the [Contribution Guidelines](CONTRIBUTING.md)
+- [ ] My code follows the [`astropy` style guidelines](http://docs.astropy.org/en/latest/development/codeguide.html) used for this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
```

Comment:
I think this template is a bit too detailed, I think I would be happy with making sure they have a good description and the 'type of change' section but I think the rest might be a bit too much (I get a bit scared/put off by detailed templates like this for other packages)